src/evaluate/visualization.py

```diff
@@ -0,0 +1,224 @@
+import textwrap
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+
+class ComplexRadar:
+    """
+    Create a complex radar chart with different scales for each variable
+    Source: https://towardsdatascience.com/how-to-create-and-visualize-complex-radar-charts-f7764d0f3652
```

Comment:
I think it would be nice to also add this to the docstring of `plot_radar` which is what we expose to the user so they can see the attribution. Otherwise one can only see it in the code.